README.md

```diff
@@ -89,10 +89,10 @@ P.S.: [We're hiring](https://mosaicml.com/jobs)!
 
 # ✍️ Citation
 ```
-@misc{mosaicml2022streaming,
```

Comment:
I think you maybe leave the year as the original publication year?

Reply:
fixed ty